src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/RepaymentPlanContentProvider.java

```diff
@@ -1,45 +1,17 @@
 package uk.gov.hmcts.cmc.claimstore.services.staff.content.countycourtjudgment;
 
 import uk.gov.hmcts.cmc.domain.models.CountyCourtJudgment;
-import uk.gov.hmcts.cmc.domain.models.ccj.PaymentOption;
 import uk.gov.hmcts.cmc.domain.models.ccj.RepaymentPlan;
 
-import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
-import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+import java.util.Optional;
 
 public final class RepaymentPlanContentProvider {
 
     private RepaymentPlanContentProvider() {
         // Utils class no constructing
     }
 
-    public static String create(CountyCourtJudgment countyCourtJudgment) {
-        switch (countyCourtJudgment.getPaymentOption()) {
-            case IMMEDIATELY:
-                return PaymentOption.IMMEDIATELY.getDescription();
-            case INSTALMENTS:
-                RepaymentPlan repaymentPlan = countyCourtJudgment.getRepaymentPlan()
-                    .orElseThrow(IllegalArgumentException::new);
-                return getRepaymentPlanContent(repaymentPlan);
-            case FULL_BY_SPECIFIED_DATE:
-                return String.format(
-                    PaymentOption.FULL_BY_SPECIFIED_DATE.getDescription(),
-                    formatDate(countyCourtJudgment.getPayBySetDate()
-                        .orElseThrow(IllegalArgumentException::new)));
-            default:
-                throw new IllegalArgumentException(
-                    "Unknown repayment type: " + countyCourtJudgment.getPaymentOption()
-                );
-        }
-    }
-
-    private static String getRepaymentPlanContent(RepaymentPlan repaymentPlan) {
-        return String.format(
-            "first payment of %s on %s. This will be followed by %s %s",
-            formatMoney(repaymentPlan.getFirstPayment()),
-            formatDate(repaymentPlan.getFirstPaymentDate()),
-            formatMoney(repaymentPlan.getInstalmentAmount()),
-            repaymentPlan.getPaymentSchedule().getDescription()
-        );
+    public static Optional<RepaymentPlan> create(CountyCourtJudgment countyCourtJudgment) {
+        return countyCourtJudgment.getRepaymentPlan();
```

Comment:
That is wrong. You basically passing raw data into PDF generator, if we wanted to do it (which we do not) we would remove `ContentProvider` completely.

To give you some background ContentProvider is an objects that reformats data for displaying in PDF's due to the fact PDF generator does not have any formatting facilitates. To give you an example we need to format date to UK format in PDF so passing raw `2018-01-01` would not be enough, in that case ContentProvider formats it to be `1 January 2018`, the form ready for being displayed in PDF.